opentelemetry-sdk/src/metrics/data/mod.rs

```diff
@@ -250,9 +250,9 @@ pub struct HistogramDataPoint<T> {
     /// The upper bounds of the buckets of the histogram.
     ///
     /// Because the last boundary is +infinity this one is implied.
-    pub bounds: Vec<f64>,
+    pub(crate) bounds: Vec<f64>,
     /// The count of each of the buckets.
-    pub bucket_counts: Vec<u64>,
+    pub(crate) bucket_counts: Vec<u64>,
 
     /// The minimum value recorded.
     pub min: Option<T>,
```

Comment:
Should we make all struct fields internal and provide getters for a consistent experience?